client/ionic/src/pages/LocalMaps.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {
+  IonContent,
+  IonHeader,
+  IonPage,
+  IonTitle,
+  IonToolbar,
+  IonItem,
+} from '@ionic/react';
+
+const LocalMaps: React.FC = () => {
+  return (
+    <IonPage>
+      <IonHeader>
```

Comment:
There's a TopNav component in 

https://github.com/WorldHealthOrganization/app/blob/master/client/ionic/src/components/TopNav.tsx

An Example implementation is at:

https://github.com/WorldHealthOrganization/app/blob/24ca3b7e1fbf28381b534835ec6e4e8fdeea8f83/client/ionic/src/pages/Menu.tsx#L9

Please refactor to include and reflect DRY.